### ISSUE TITLE
How to deal with empty fields?

### ISSUE THREAD

**Issue by NielsNicolai** [Big-Life-Lab/covid-19-wastewater#84]
When filling out the Ontario Template, one thing I noticed is that people will have different ways of handling missing or non-existing data. This is especially important when having all these `other` labels which might be left empty if an option exists in the dropdown menu. How should the user ideally deal with that? Is this relevant to take up into a guideline of completing the data model?

Different options for non-existing data in the `...Other` variables might be:
- `(empty)`
- `None`
- `-`
- `NaN`

**Comment by DougManuel** [comment]
Good question. Missing data is always challenging for input forms that have manual input. 

What about making a suggestion for missing data? @NielsNicolai do you have a suggested format. I lean to `NA`.

Ultimately, missing data can be handled during data cleaning, but another good reason to suggest a format so that data users and data scientist can be consistent.

**Comment by hswerdfe** [comment]
I would say in all cases missing information should be blank, most import programs will interpret a blank as NA in both numeric and text columns, while an NA in a numeric column can cause issues with import, in some cases.

ultimately it does not matter though

**Comment by NielsNicolai** [comment]
I don't agree with a blank value. People might just have forgotten to fill out the field.
`NA` is fine for me.

**Comment by vipileggi** [comment]
I think we need to communicate the rules so everyone is clear. Also we should consider reducing the burden on end users I think once the rules are clear using 'post processing' is one sure way to ensure consistency. In the case of the 'other' column leaving it blank makes sense if its not used. In the case of numeric values a blank or NA may be appropriate if the sample analysis was not done.

**Comment by NielsNicolai** [comment]
Good point @vipileggi We need to have some rule on this.

Note that my problem refers not to the `other` columns, but rather to measurements that should be there but for some reason they are not. 

Imagine that for every day of the week you have a daily average `TSS` concentration at your sampling WWTP. However, due to operational failure (sensor problems, storm events, ...), the TSS measurement is not giving the correct value for one day. What do you do?

It is my opinion that this kind of missing data should be included, and annotated as to why they are missing using the `notes` variable.

**Comment by vipileggi** [comment]
@NielsNicolai yes, agreed,; having the information (and its not misused) is always better. But this also points to starting to write down the rules we can all agree on for implement for V1,1,0 if at all possible.  

@DougManuel  @hswerdfe  @NielsNicolai At MECP with the data we received so far from 3-labs we've started to classify and propose some validation solutions but will start to right down the rules for discuss here as well. Likely will post our interim solution by Monday. Busy the next couple of days. Thanks

**Comment by NielsNicolai** [comment]
Add a description of how to handle a missing value in the description of the `value` field, and link to the `notes` field.

**Comment by NielsNicolai** [comment]
Solved with this [commit](https://github.com/Big-Life-Lab/covid-19-wastewater/commit/96e5f723bda8826ca9546e63f8fa93aec4928c3f)
